### PR TITLE
fix institute settings bug

### PR DESCRIPTION
**Pull request by northwestwitch**
This PR is a hotfix for institute settings page that doesn't work

**How to test**:
1. test on scout stage

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN
- [x] tests executed by CR

### CODE REVIEW

File: scout/server/blueprints/institutes/views.py [hunk 1]
```diff
@@ -65,8 +65,8 @@ def institute(institute_id):
     # get all other institutes to populate the select of the possible collaborators
     institutes_tuples = []
     for inst in store.institutes():
-        if not inst['internal_id'] == institute_id:
-            institutes_tuples.append( ((inst['internal_id'], inst['internal_id']) ))
+        if not inst['_id'] == institute_id:
+            institutes_tuples.append( ((inst['_id'], inst['_id']) ))
```
Comment (dnil) [inline scout/server/blueprints/institutes/views.py]: :+1:

**Comment by northwestwitch** [comment]
Took me a while to understand that I had to manually restart the server. Works!:

image

**Comment by dnil** [comment]
Wait, what about the institutes tuples? Shouldn't there be like a display_id shown from it?

**Comment by dnil** [comment]
display_name

**Comment by northwestwitch** [comment]
> Wait, what about the institutes tuples? Shouldn't there be like a display_id shown from it?

I see a bug when you choose institutes to share with, I don't know if it's the same

**Comment by dnil** [comment]
think so..

**Comment by northwestwitch** [comment]
It says 404 not found

**Comment by dnil** [comment]
Oh, and there are a couple of direct links into the institute page on the case page that need changing:

case.html line 558
```
        <a href="{{ url_for('overview.institute', institute_id=institute.internal_id) }}">Add options</a>
```
and 
line 749
```
            <a href="{{ url_for('overview.institute', institute_id=institute.internal_id) }}">Add options</a>```

**Comment by northwestwitch** [comment]
Fixed!

**Pull request merged**